asn1/InterledgerProtocol.asn

```diff
@@ -17,7 +18,17 @@ InterledgerProtocolPayment ::= SEQUENCE {
     amount UInt64,
     -- Destination ILP Address
     account Address,
-    -- Information for recipient (transport layer information)
+    -- Information for recipient (application layer information)
+    data OCTET STRING (SIZE (0..32767)),
+    -- Enable ASN.1 Extensibility
+    extensions SEQUENCE {
+        ...
+    }
+}
+
+InterledgerProtocolFulfillment ::= SEQUENCE {    
+    fulfillment UInt256,
+    -- Information for recipient (application layer information)
     data OCTET STRING (SIZE (0..32767)),
```

Comment:
The possibility of sending `data` back in some kind of payment response is interesting. Do you have any use case in mind? Another question that arises is why does `data` have to travel back through ILP as opposed to more efficient out-of-band communication?